configs/eslint-plugin-compass/package.json

```diff
@@ -34,7 +34,7 @@
     "test-cov": "nyc --compact=false --produce-source-map=false -x \"**/*.spec.*\" --reporter=lcov --reporter=text --reporter=html npm run test",
     "test-watch": "npm run test -- --watch",
     "test-ci": "npm run test-cov",
-    "reformat": "npm run prettier -- --write . && npm run eslint . --fix"
+    "reformat": "npm run eslint . -- --fix && npm run prettier -- --write ."
```

Comment:
Otherwise prettier fix can be messed up by eslint autofix, so you kinda have to run it again